common/utils/category_resolver.py

```
@@ -0,0 +1,43 @@
+from API.models import Insult
+from loguru import logger
+from prometheus_client import Histogram
+
+
+class Resolver:
+    cat_resolver_metric = Histogram(
+        "category_resolver", "Metric of the Duration of categories in request"
+    )
+
+    @classmethod
+    def _generate_insult_category_dict(cls) -> Dict[str, str]:
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)